src/System.Azure.Experimental/System.Azure.Experimental.csproj

```diff
@@ -8,15 +8,17 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Description>Tensor class which represents and extends multi-dimensional arrays</Description>
+    <Description>Set of building blocks for Azure SDK</Description>
     <TargetFramework>netstandard1.3</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PackageTags>BCL for Azure SDK </PackageTags>
     <!-- disable automatic compile item inclusion so that we can set the correct metadata on TextTemplate-generated sources -->
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
We don't need this attribute and hence can get rid of this and the explicit list of the *.cs files to include below.